pandas/tests/arithmetic/conftest.py

```diff
@@ -240,5 +240,14 @@ def box_with_array(request):
     return request.param
 
 
+@pytest.fixture(params=[pd.Index, pd.Series, tm.to_array, np.array, list], ids=id_func)
+def box_1d_array(request):
+    """
+    Fixture to test behavior for Index, Series, pandas Array, numpy Array and list
```

Comment:
tm.to_array is not the same as pd.array

Reply:
Changed fixture doc from pandas array to tm.to_array